app/_src/mesh/plan-and-deploy/license.md

```diff
@@ -124,6 +124,24 @@ In Universal mode, configure a valid license by using the following environment
 * `KMESH_LICENSE_PATH` - value with the path to a valid license file.
 * `KMESH_LICENSE_INLINE` - value with the actual contents of the license file.
 
+## Update a license
+
+### Kubernetes
+
+1. Update the license 
+  Update the `kong-mesh-license` Secret in `kong-mesh-system` namespace with a new license.
+1. Restart the control plane
+  ```sh
+  kubectl rollout restart -n kong-mesh-system deployment kong-mesh-control-plane
+  ```
+
+### Universal
+
+1. Update the license
+  If you used `KMESH_LICENSE_PATH`, update the content of the file.
+  If you used `KMESH_LICENSE_INLINE`, update the value of the environemnt variable.
```

Comment:
```suggestion
1. Update the license by doing one of the following:
  - If you used `KMESH_LICENSE_PATH`, update the content of the `values.yaml` file.
  - If you used `KMESH_LICENSE_INLINE`, update the value of the environment variable in the `values.yaml` file.
```
would this be accurate?

Reply:
Not really, there is no `values.yaml` in Universal mode because it's helm.